Add workload section to Benchmarks (#4705)
opensearch-project/documentation-website: _benchmark/workloads/indices.md

---
layout: default
title: indices
parent: Workload reference
nav_order: 65
---

The `indices` element contains a list of all indices used in the workload. 

## Example

```json
"indices": [
    {
      "name": "geonames",
      "body": "geonames-index.json",
    }
]
```

## Configuration options

Use the following options with `indices`:

Parameter | Required | Type | Description
:--- | :--- | :--- | :---
| `name` | Yes | String | The name of the index template. |

> | `body` | No | String | The file name corresponding to the index definition that is used in the body of the create index API. |

Reviewer: ```suggestion
| `body` | No | String | The file name corresponding to the index definition used in the body of the create index API. |
```

> | `body` | No | String | The file name corresponding to the index definition that is used in the body of the create index API. |

Reviewer: Should the name of the API be capitalized?

Author: Done.

> | `body` | No | String | The file name corresponding to the index definition that will be used in the body of the create index API. |

Reviewer 2: ```suggestion
| `body` | No | String | The file name corresponding to the index definition that is used in the body of the create index API. |
```
